common/djangoapps/student/views.py

```diff
@@ -735,6 +736,8 @@ def dashboard(request):
         'course_programs': course_programs,
         'disable_courseware_js': True,
         'xseries_credentials': xseries_credentials,
+        'use_ecommerce_payment_flow': EcommerceService().is_enabled(),
```

Comment:
Better to set to a variable and reference that twice, versus building up the object twice.